server/controllers/messages.coffee

```diff
@@ -194,12 +196,30 @@ module.exports.send = (req, res, next) ->
     delete req.body.isDraft
 
     message = req.body
-    message.html = message.html.trim() if message.composeInHTML
+    if message.html
+        minifierOpts =
```

Comment:
Please move static tool option at the beginning of file  or to another file. 

(Its not a big deal in this case, but will cause performance issues if moved to the render cycle in a refactoring)